src/Nethermind/Nethermind.Network.Stats/Model/Node.cs

```diff
@@ -69,6 +70,7 @@ public string ClientId
 
         public string EthDetails { get; set; }
         public long CurrentReputation { get; set; }
+        public Dictionary<string, int> AgreedCapability { get; set; } = new();
```

Comment:
Not sure if I like it.

Reply:
+1 we don't need it, we can have the this info from already existing `EthDetails`

Reply:
We don't have the version in the existing `EthDetails`. Are you sure?

Reply:
Ow, there is already a `ProtocolVersion`, nevermind.